script/proto/diagnostic.lua

```diff
@@ -171,6 +171,14 @@ m.register {
     status   = 'Any',
 }
 
+m.register {
+    'global-element',
+} {
+    group   = 'conventions',
+    severity = 'Information',
```

Comment:
Do we want that to be an information only?
I'd opt for "Warning" if that is available.